hf_vision_model_tfserving_gke/deploy.sh

```diff
@@ -0,0 +1,119 @@
+#! /bin/sh
+# This shell script apply Deployment to the designated GKE cluster
+# Prerequisite: You should provision GKE cluster beforehand
+
+## 0. Define Variables
+GCP_PROJECT_ID=gcp-ml-172005
+### Path where service account key file is (JSON)
+SA_KEY_FILE=/tmp/gcp-ml-172005-5c6ccb3685b9.json
```

Comment:
Do we still need it?

I mean someone has already executed GCP auth, then it shouldn't be needed, no?